src/components/Trends/TrendsPanel/TrendsPanel.scss

```diff
@@ -87,8 +89,6 @@
         }
 
         ul.tooltip-ul {
-          list-style: none;
```

Comment:
I think this rule needs to stay

```list-style: none;```

I'm seeing bullets next to the legend in trends panel:
<img width="384" alt="Screenshot 2024-10-22 at 9 15 44 AM" src="https://github.com/user-attachments/assets/c5bef186-99d8-4cee-9134-88ef2a0e17f7">

Reply:
I added this back. I was initially going to add the `.m-list--unstyled`  class to the TrendsPanel component,  but didnt see anywhere to put it, so this will also work.